src/packages/@teams/features/team-explorer/components/workspace-list-view/WorkspaceListView.svelte

```diff
@@ -0,0 +1,202 @@
+<script lang="ts">
+  import {
+    DoubleLeftIcon,
+    LeftIcon,
+    RightIcon,
+    DoubleRightIcon,
+  } from "$lib/assets/app.asset";
+  import type { TeamDocument } from "@app/database/database";
+  import { calculateTimeDifferenceInDays } from "$lib/utils/workspacetimeUtils";
+  import Table from "../../../../../@deprecate/components/table/Table.svelte";
+  import Rows from "../../../../../@deprecate/components/dashboard/rows/Rows.svelte";
```

Comment:
not required to import this component from $lib that represents deprecate. please push this component to newer folder structure

Reply:
same